tools/PlateManager/Program.cs

```diff
@@ -37,7 +38,7 @@ static Task<int> Main(string[] args)
             return root.InvokeAsync(args);
         }
 
-        static async Task Run(IEnumerable<FileInfo> file, Uri storage, bool interactive, string baseUrl)
+        static async Task Run(IEnumerable<FileInfo> file, Uri storage, bool interactive, string baseUrl, bool doNotSkipExisting)
```

Comment:
I think "doNotSkipExisting" is less clear than "skipExisting", consider flipping the directionality of this variable

Reply:
Fixed